packages/office-addin-debugging/src/start.ts

```diff
@@ -35,6 +35,15 @@ export async function isDevServerRunning(port: number): Promise<boolean> {
     return isRunning;
 }
 
+export async function isDevBuildComplete(sourceLocation: string): Promise<boolean> {
```

Comment:
You could name this to reflect what it is actually doing, which is just seeing if the specified sourceLocation url can be fetched. You could make the parameter name simply "url", and call it isUrlAvailable().

Reply:
made change locally